### PR TITLE
docs: sync Chinese benchmark doc and add notes on running the benchmark

### LINKED ISSUE #5966: doc: benchmark results is not so good,what is the problem,help
### Issue description

I have see  the  doc  about the benchmark，the result is good： https://apisix.apache.org/docs/apisix/benchmark/
I try it myself， but  it is not so  good，any one  who can tell me  how to  optimize it.
Here is my  env:
I  have three host, and  they are all  4c8g
host13: I deploy  apisix2.11.0  use the source code
host14: I use it  to run wrk, the  HTTP benchmarking tool
host15: I deploy a nginx, as a upstream, and k8s, I also deploy a apisix  as a pod  in  this host

I have not use any plugins, and only config the nginx  as a upstream of the apisix. And the nginx return a string directory

the test result like this:
wrk  directory call the nginx, qps is 30217
wrk call the host13 apisix,  qps is 11003.17
wrk call the host15 apisix, qps is  8002.80
here is the wrk  detail:
[root@k8s_test1 data]# ./wrk -t200 -c5000 -d30s  http://172.16.11.15:1980/hello
Running 30s test @ http://172.16.11.15:1980/hello
  200 threads and 5000 connections
  Thread Stats   Avg      Stdev     Max   +/- Stdev
    Latency   215.86ms  235.20ms   2.00s    88.07%
    Req/Sec   153.61     98.81     1.50k    71.78%
  912979 requests in 30.21s, 302.13MB read
  Socket errors: connect 0, read 4, write 163, timeout 1100
Requests/sec:  30217.23
Transfer/sec:     10.00MB

[root@k8s_test1 data]# ./wrk -t200 -c5000 -d30s  http://172.16.11.13:9080/hello
Running 30s test @ http://172.16.11.13:9080/hello
  200 threads and 5000 connections
  Thread Stats   Avg      Stdev     Max   +/- Stdev
    Latency   449.90ms  298.54ms   2.00s    81.53%
    Req/Sec    55.02     32.43   420.00     67.66%
  332622 requests in 30.23s, 110.40MB read
  Socket errors: connect 0, read 18, write 263, timeout 3732
  Non-2xx or 3xx responses: 548
Requests/sec:  11003.17
Transfer/sec:      3.65MB

[root@k8s_test1 data]# ./wrk -t200 -c5000 -d30s  http://172.16.11.15:30980/hello
Running 30s test @ http://172.16.11.15:30980/hello
  200 threads and 5000 connections
  Thread Stats   Avg      Stdev     Max   +/- Stdev
    Latency   109.96ms   89.03ms   2.00s    93.34%
    Req/Sec   100.90     92.24   717.00     76.64%
  242268 requests in 30.27s, 80.39MB read
  Socket errors: connect 0, read 2712, write 14878, timeout 2282
Requests/sec:   8002.80
Transfer/sec:      2.66MB

### Environment

- apisix version (cmd: `apisix version`):2.11.0
- OS (cmd: `uname -a`):centos 7
- OpenResty / Nginx version (cmd: `nginx -V` or `openresty -V`):
- etcd version, if have (cmd: run `curl http://127.0.0.1:9090/v1/server_info` to get the info from server-info API):
- apisix-dashboard version, if have:
- the plugin runner version, if the issue is about a plugin runner (cmd: depended on the kind of runner):
- luarocks version, if the issue is about installation (cmd: `luarocks --version`):

### PR DESCRIPTION
**Pull request by yunwei37**
### Description

1. As described in the issue: https://github.com/apache/apisix/issues/5966#issuecomment-1006105454, I added @membphis's comment to the Benchmark page;

2. It seems currently we have both English version and Chinese version doc about benchmark:

   - https://github.com/apache/apisix/blob/master/docs/en/latest/benchmark.md
   - https://github.com/apache/apisix/blob/master/docs/zh/latest/benchmark.md
      but the Chinese doc is missing some parts, so I translated them from the English version;

Fixes #5966

### Checklist

- [x] I have explained the need for this PR and the problem it solves
- [x] I have explained the changes or the new features added to this PR
- [x] I have added tests corresponding to this change
- [x] I have updated the documentation to reflect this change
- [x] I have verified that this change is backward compatible (If not, please discuss on the [APISIX mailing list](https://github.com/apache/apisix/tree/master#community) first)

### CODE REVIEW

**Comment by juzhiyuan** [comment]
Thanks for your contribution! cc @navendu-pottekkat @hf400159 to have a review when you have time :)

File: docs/zh/latest/benchmark.md [hunk 1]
```diff
@@ -49,6 +49,29 @@ title: 压力测试
 
 ![flamegraph-1](../../assets/images/flamegraph-1.jpg)
 
+如果您想在您的机器上运行基准测试，您应该同时运行另一个 Nginx 来监听 80 端口：
```
Comment (SylviaBABY) [inline docs/zh/latest/benchmark.md:52]:
```suggestion
如果想在机器上运行基准测试，你应该同时运行另一个 NGINX 来监听 80 端口：
```

Comment (SylviaBABY) [reply docs/zh/latest/benchmark.md:52]: For [中文文档写作指南](https://apisix.apache.org/zh/docs/general/documentation-style-guide/), you can change `您` to `你`, and `Nginx` to `NGINX`. Pls check all situation contents^^

File: docs/zh/latest/benchmark.md [hunk 2]
```diff
@@ -69,3 +92,47 @@ title: 压力测试
...
+
+**注意**: 如果您想运行大量连接的基准测试，例如：
+
+```shell
+./wrk -t200 -c5000 -d30s http://127.0.0.1:9080/hello
+```
+
+您可能需要更新 [conf/config-default.yaml](https://github.com/apache/apisix/blob/master/conf/config-default.yaml#L242) 中的 **keepalive** 配置项。
+
+超过配置数量的连接将成为短连接。更多文档可以参考： http: //nginx.org/en/docs/http/ngx_http_upstream_module.h
```
Comment (SylviaBABY) [inline docs/zh/latest/benchmark.md:138]:
```suggestion
超过配置数量的连接将成为短连接。更多文档可以参考：http://nginx.org/en/docs/http/ngx_http_upstream_module.html
```

File: docs/en/latest/benchmark.md [hunk 3]
```diff
@@ -126,3 +126,15 @@ then run wrk:
...
+
+**Note**: if you want to run the benchmark with a large number of connections, for example:
+
+```shell
+./wrk -t200 -c5000 -d30s http://127.0.0.1:9080/hello
+```
+
+You may have to update the **keepalive** config in the [conf/config-default.yaml](https://github.com/apache/apisix/blob/master/conf/config-default.yaml#L242).
+
+Connections exceeding this number will become short connections. For more doc: http://nginx.org/en/docs/http/ngx_http_upstream_module.h
```
Comment (SylviaBABY) [inline docs/en/latest/benchmark.md:140]:
```suggestion
Connections exceeding this number will become short connections. For more details you can refer to: http://nginx.org/en/docs/http/ngx_http_upstream_module.html
```

**Comment by yunwei37** [comment]
@SylviaBABY Thank you for reviewing! These should be fixed now.

File: docs/zh/latest/benchmark.md [hunk 4]
```diff
@@ -49,6 +49,29 @@ title: 压力测试
 
 ![flamegraph-1](../../assets/images/flamegraph-1.jpg)
 
+如果想在机器上运行基准测试，你应该同时运行另一个 NGINX 来监听 80 端口：
```
Comment (hf400159) [inline docs/zh/latest/benchmark.md:52]:
```suggestion
如果你想在你的服务器上运行基准测试，你需要同时启动两个 NGINX 实例并监听 `80` 端口。

你可以通过以下命令创建一个路由并添加已经启动的 NGINX 服务：
```

File: docs/zh/latest/benchmark.md [hunk 5]
```diff
@@ -69,3 +92,47 @@ title: 压力测试
 
 火焰图的采样结果：
 ![火焰图采样结果](../../assets/images/flamegraph-2.jpg)
+
+如果想在机器上运行基准测试，你应该同时运行另一个 NGINX 来监听 80 端口：
```
Comment (hf400159) [inline docs/zh/latest/benchmark.md:96]:
```suggestion
如果你想在本地服务器上运行基准测试，你需要同时启动两个 NGINX 实例并监听 `80` 端口，并安装 [wrk](https://github.com/wg/wrk/)。

你可以通过以下命令创建一个路由并添加已经启动的 NGINX 服务：
```

File: docs/en/latest/benchmark.md [hunk 6]
```diff
@@ -126,3 +126,15 @@ then run wrk:
 ```shell
 wrk -d 60 --latency http://127.0.0.1:9080/hello
 ```
+
+For more reference on how to run the benchmark test, you can see this [PR](https://github.com/apache/apisix/pull/6136) and this [script](https://gist.github.com/membphis/137db97a4bf64d3653aa42f3e016bd01).
+
+**Note**: if you want to run the benchmark with a large number of connections, for example:
+
+```shell
```
Comment (hf400159) [inline docs/en/latest/benchmark.md:134]:
```suggestion
```bash
```

File: docs/zh/latest/benchmark.md [hunk 7]
```diff
@@ -49,6 +49,29 @@ title: 压力测试
 
 ![flamegraph-1](../../assets/images/flamegraph-1.jpg)
 
+如果想在机器上运行基准测试，你应该同时运行另一个 NGINX 来监听 80 端口：
+
+```shell
```
Comment (hf400159) [inline docs/zh/latest/benchmark.md:54]:
```suggestion
```bash
```

File: docs/zh/latest/benchmark.md [hunk 8]
```diff
@@ -49,6 +49,29 @@ title: 压力测试
...
+            "127.0.0.2:80": 1
+        }
+    }
+}'
+```
+
+然后运行 wrk：
+
+```shell
+wrk -d 60 --latency http://127.0.0.1:9080/hello
```
Comment (hf400159) [inline docs/zh/latest/benchmark.md:72]:
```suggestion
./wrk -d 60 --latency http://127.0.0.1:9080/hello
```

File: docs/zh/latest/benchmark.md [hunk 9]
```diff
@@ -69,3 +92,47 @@ title: 压力测试
...
+```
+
+有关如何运行基准测试的更多参考，你可以查看此[PR](https://github.com/apache/apisix/pull/6136)和此[脚本](https://gist.github.com/membphis/137db97a4bf64d3653aa42f3e016bd01)。
+
+**注意**: 如果你想运行大量连接的基准测试，例如：
```
Comment (hf400159) [inline docs/zh/latest/benchmark.md:130]:
```suggestion
:::tip

如果你想测试大量连接的基准测试，你可能需要更新 [`./conf/config-default.yaml`](https://github.com/apache/apisix/blob/master/conf/config-default.yaml#L242) 中的 **keepalive** 配置项，否则超过配置数量的连接将成为短连接。

如果你需要了解更多信息，请参考：[ngx_http_upstream_module] http://nginx.org/en/docs/http/ngx_http_upstream_module.html)。

:::
```

File: docs/zh/latest/benchmark.md [hunk 10]
```diff
@@ -69,3 +92,47 @@ title: 压力测试
...
+
+```shell
+./wrk -t200 -c5000 -d30s http://127.0.0.1:9080/hello
+```
+
+你可能需要更新 [conf/config-default.yaml](https://github.com/apache/apisix/blob/master/conf/config-default.yaml#L242) 中的 **keepalive** 配置项。
+
+超过配置数量的连接将成为短连接。更多文档可以参考：[ngx_http_upstream_module](http://nginx.org/en/docs/http/ngx_http_upstream_module.html)
```
Comment (hf400159) [inline docs/zh/latest/benchmark.md:138]:
```suggestion
```

File: docs/zh/latest/benchmark.md [hunk 11]
```diff
@@ -69,3 +92,47 @@ title: 压力测试
...
+
+```shell
+wrk -d 60 --latency http://127.0.0.1:9080/hello
+```
+
+有关如何运行基准测试的更多参考，你可以查看此[PR](https://github.com/apache/apisix/pull/6136)和此[脚本](https://gist.github.com/membphis/137db97a4bf64d3653aa42f3e016bd01)。
+
+**注意**: 如果你想运行大量连接的基准测试，例如：
+
+```shell
```
Comment (hf400159) [inline docs/zh/latest/benchmark.md:132]:
```suggestion
你可以使用以下命令运行大量连接的基准测试：

```bash
```

File: docs/zh/latest/benchmark.md [hunk 12]
```diff
@@ -69,3 +92,47 @@ title: 压力测试
...
+        "type": "roundrobin",
+        "nodes": {
+            "127.0.0.1:80": 1,
+            "127.0.0.2:80": 1
+        }
+    }
+}'
+```
+
+然后运行 wrk：
```
Comment (hf400159) [inline docs/zh/latest/benchmark.md:122]:
```suggestion
完成配置后，可以使用以下命令进行测试：
```

**Review by hf400159 — COMMENTED** [review]
For the English version, please refer to the Chinese version for revision.

File: docs/zh/latest/benchmark.md (hunk 4 above)
Comment (yunwei37) [reply docs/zh/latest/benchmark.md:52]: Changing this sentence may lead to some ambiguity: is one NGINX listening on port 80 or both? Also, one of the two NGINXs should be started by the APISIX command and does not require an additional manual start.

修改这句话可能会带来一些歧义：到底是一个 NGINX 在监听 80 端口还是两个都在监听 80 端口呢？另外这两个 NGINX 其中一个 NGINX 应该是由 APISIX 命令启动的，并不需要额外手动启动一次。

File: docs/en/latest/benchmark.md [hunk 13]
```diff
@@ -126,3 +126,15 @@ then run wrk:
 ```shell
 wrk -d 60 --latency http://127.0.0.1:9080/hello
```
[the remark this answers is not in the document]
Comment (yunwei37) [reply docs/en/latest/benchmark.md:127]: The English version is using this command as well. I think the `wrk` binary is not always in the current directory, so only `wrk`  is better?

File: docs/zh/latest/benchmark.md (hunk 8 above)
Comment (yunwei37) [reply docs/zh/latest/benchmark.md:72]: Described above.

File: docs/zh/latest/benchmark.md [hunk 14]
```diff
@@ -69,3 +92,47 @@ title: 压力测试
...
+
+```shell
+wrk -d 60 --latency http://127.0.0.1:9080/hello
```
[the remark this answers is not in the document]
Comment (yunwei37) [reply docs/zh/latest/benchmark.md:125]: Described above.

File: docs/en/latest/benchmark.md (hunk 13 above)
[the remark this answers is not in the document]
Comment (hf400159) [reply docs/en/latest/benchmark.md:127]: ok, Please be consistent.

File: docs/zh/latest/benchmark.md [hunk 15]
```diff
@@ -69,3 +92,49 @@ title: 压力测试
...
+
+:::tip
+
+如果你想测试大量连接的基准测试，你可能需要更新 [`./conf/config-default.yaml`](https://github.com/apache/apisix/blob/master/conf/config-default.yaml#L242) 中的 **keepalive** 配置项，否则超过配置数量的连接将成为短连接。你可以使用以下命令运行大量连接的基准测试：
+
+```bash
+wrk -t200 -c5000 -d30s http://127.0.0.1:9080/hello
+```
+
+如果你需要了解更多信息，请参考：[ngx_http_upstream_module] http://nginx.org/en/docs/http/ngx_http_upstream_module.html)。
```
Comment (hf400159) [inline docs/zh/latest/benchmark.md:138]:
```suggestion
如果你需要了解更多信息，请参考：[ngx_http_upstream_module]http://nginx.org/en/docs/http/ngx_http_upstream_module.html)。
```

File: docs/zh/latest/benchmark.md [hunk 16]
```diff
@@ -69,3 +92,49 @@ title: 压力测试
 
 火焰图的采样结果：
 ![火焰图采样结果](../../assets/images/flamegraph-2.jpg)
+
+如果想在机器上运行基准测试，你应该同时运行另一个 NGINX 来监听 80 端口：
```
Comment (hf400159) [inline docs/zh/latest/benchmark.md:96]: 如果你需要在本地服务器上运行基准测试，你需要同时运行另一个 NGINX 实例来监听 80 端口：

File: docs/zh/latest/benchmark.md [hunk 17]
```diff
@@ -49,6 +49,29 @@ title: 压力测试
 
 ![flamegraph-1](../../assets/images/flamegraph-1.jpg)
 
+如果想在机器上运行基准测试，你应该同时运行另一个 NGINX 来监听 `80` 端口：
```
Comment (hf400159) [inline docs/zh/latest/benchmark.md:52]:
```suggestion
如果你需要在本地服务器上运行基准测试，你需要同时运行另一个 NGINX 实例来监听 80 端口：
```

File: docs/zh/latest/benchmark.md (hunk 15 above)
Comment (hf400159) [reply docs/zh/latest/benchmark.md:138]: extra spaces.

**Review by hf400159 — APPROVED** [review]

**Review by juzhiyuan — APPROVED** [review]

**Pull request merged**